### ISSUE TITLE
taillog.php type mismatch

### ISSUE THREAD

**Issue by dschaper** [pi-hole/AdminLTE#504]
**In raising this issue, I confirm the following (please check boxes, eg [X] - no spaces) Failure to fill the template will close your issue:**

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*
- [x] The issue I'm reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the codebase?:**

10

---
**[BUG] Actual Behaviour:**

From: https://discourse.pi-hole.net/t/lost-connection-to-api-centos7/3078/5

PHP Warning error filling disk with lighttpd error.log, repeating:
```
2017-05-11 15:18:32: (mod_fastcgi.c.2543) FastCGI-stderr: PHP Warning:  feof() expects parameter 1 to be resource, boolean given in /var/www/html/admin/scripts/pi-hole/php/tailLog.php on line 32
2017-05-11 15:18:32: (mod_fastcgi.c.2543) FastCGI-stderr: PHP Warning:  fgets() expects parameter 1 to be resource, boolean given in /var/www/html/admin/scripts/pi-hole/php/tailLog.php on line 33
2017-05-11 15:18:32: (mod_fastcgi.c.2543) FastCGI-stderr: PHP Warning:  feof() expects parameter 1 to be resource, boolean given in /var/www/html/admin/scripts/pi-hole/php/tailLog.php on line 32
2017-05-11 15:18:32: (mod_fastcgi.c.2543) FastCGI-stderr: PHP Warning:  fgets() expects parameter 1 to be resource, boolean given in /var/www/html/admin/scripts/pi-hole/php/tailLog.php on line 33
```

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

**Comment by Mukrosz** [comment]
Thanks for posting. Unfortunately I cannot run pihole -d. I run out of space quicker than pihole -d finishes

**Comment by Mukrosz** [comment]
Thanks dschaper,
I stopped lighttpd and ran pihole -d
Token: 3fj3x8jvae

**Comment by Mcat12** [comment]
@Mukrosz If you're brave, you can try testing those changes by editing `/var/www/html/admin/scripts/pi-hole/php/tailLog.php`.

**Comment by DL6ER** [comment]
Was marked "Fixed in Next Release" 19 days ago. We released Pi-hole v3.1 9 days ago, so I assume it is fixed by now.
